apps/api/src/workspace/workspace.e2e.spec.ts

```diff
@@ -756,7 +756,9 @@ describe('Workspace Controller Tests', () => {
           Authority.READ_VARIABLE,
           Authority.READ_WORKSPACE
         ],
-        projectSlugs: [project2Response.slug]
+        projectEnvironments: [
+          { projectSlug: project2Response.slug, environmentSlugs: [] }
```

Comment:
Perhaps we should include the environment slugs so that the other tests pass successfully. this will ensure that the change in logic that i talked about previously works fine